ihontaryk/homework_8/person.py

```diff
@@ -0,0 +1,67 @@
+from datetime import datetime, date
+
+
+class Person:
+    """
+    Class for persons
+    """
+
+    count = 0
+
+    def __init__(self, first_name, last_name, dob, country, city, phone, job_title, experience, salary,
+                 gender='unknown'):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.dob = datetime.strptime(dob, "%d.%m.%Y")
+        self.gender = gender
+        self.country = country
+        self.city = city
+        self.phone = phone
+        self.job_title = job_title
+        self.experience = float(experience)
+        self.salary = float(salary)
+        self.__class__.count += 1
+
+    def get_full_name(self):
+        return self.first_name + ' ' + self.last_name
+
+    def get_age(self):
+        not_full_year = (self.dob.month, self.dob.day) > (date.today().month, date.today().day)
+        age = (date.today().year - self.dob.year) - not_full_year
+        return age
+
+    def show_personal_info(self):
+        return f'Full name: {self.get_full_name()}, Age: {self.get_age()}, Gender: {self.gender}'
+
+    def show_contacts(self):
+        return f'Country: {self.country}, City: {self.city}, Phone: {self.phone}'
+
+    def show_job_profile(self):
+        return f"Job title: {self.job_title}, Experience: {self.experience}, Salary: {self.salary} USD"
```

Comment:
show_   is about representation, but this class purpose is to store and calculate
do not mix responsibilities
and these methods are badly structured, so they do not make much sense somewhere except tests
and preparation of functionality purely for tests is a bad practice